.github/workflows/test.yml

```diff
@@ -25,7 +25,7 @@ on:
 
 env:
   environment: "qa"
-  timezone: "Europe/London"
+  TIMEZONE: ${{ vars.TIMEZONE }}
```

Comment:
I know GHA variables are case insensitive, but I think it would be nicer if `timezone` was lowercase here for consistency.